### ISSUE TITLE
Search contacts and bookmarks

### ISSUE THREAD

**Issue by adamcarter81** [organicmaps/organicmaps#7309]
When using on the phone (Android) or Android Auto - it would be useful if I could search contact names and also bookmarks. E.g I have saved some bookmarks, and when searching I think it would be great if these could be shown in the results above the global search results

**Comment by biodranik** [comment]
You're speaking about the general search field, where contacts and bookmarks can also be searched/displayed as search results, right? That is a good idea, I think there are already some related issues about it.

**Comment by oKtosiTe** [comment]
For me, ideally exact bookmark name matches would be prioritized over general map search results.

**Comment by adamcarter81** [comment]
> For me, ideally exact bookmark name matches would be prioritized over general map search results.

Yes exactly this, thanks for the explanation - this is especially useful when using in AA and searching. It seems more intuitive to have a search that can search bookmarks, rather than needing to exit search and go to bookmarks

**Comment by Mikaela** [comment]
I think this is partially a duplicate or at least related to https://github.com/organicmaps/organicmaps/issues/3474

**Comment by guplem** [comment]
This too: https://github.com/organicmaps/organicmaps/issues/808

**Comment by hhartzer** [comment]
Should we close this out in favor of these two?

#3474
#808

Not really helpful to have duplicate open issues. I think this is a great idea, though!

**Comment by oKtosiTe** [comment]
> Should we close this out in favor of these two?

I can't speak for @adamcarter81 of course, but I am okay with that.
